mlflow/pyfunc/mlserver.py

```diff
@@ -7,8 +8,15 @@
 
 
 def get_cmd(
-    model_uri: str, port: int = None, host: str = None, nworkers: int = None
+    model_uri: str, port: int = None, host: str = None, nworkers: int = None,
+    conda_env_root_dir: str = None
 ) -> Tuple[str, Dict[str, str]]:
+    if conda_env_root_dir is not None:
+        raise MlflowException(
+            "MLServer does not support setting `conda_env_root_dir`",
+            error_code=INVALID_PARAMETER_VALUE,
+        )
```

Comment:
How hard is it to support this? Is https://github.com/mlflow/mlflow/pull/5561/files#diff-e607d3e6aefa036b1bfe9416fd1aa8b0934120f458cf263ddebab224f832e4a5R389-R390 sufficient?

Reply:
The "get_cmd" function does not need the "conda_env_root_dir" argument, because in `backend.serve` it calls `_execute_in_conda_env` with env_root_dir setting. Removed the argument for both mlserver and scoring_server.